orderapi/src/main/java/fi/hel/verkkokauppa/order/api/data/OrderItemMetaDto.java

```diff
@@ -17,5 +17,19 @@
     private String visibleInCheckout;
     private String ordinal;
 
+    public OrderItemMetaDto() {
+    }
+
+    public OrderItemMetaDto(String orderItemId, String orderId, String key, String value, String label,
```

Comment:
Should use lombok @AllArgsConstructor? No need to create constructors?

Reply:
Yes, and it was missing the metaItemId parameter too.